src/components/Cover/index.js

```diff
@@ -137,12 +173,23 @@ export const Cover = ({
 
     {caption && !captionUrl ? <Caption>{caption}</Caption> : null}
 
-    <BackgroundImage src={imageUrl} blurred={blurred} tinted={tinted} />
+    <BackgroundImage
+      src={imageUrl}
+      srcSet={images}
+      blurred={blurred}
+      tinted={tinted}
+    />
   </Container>
 )
 
 Cover.propTypes = {
-  imageUrl: PropTypes.string.isRequired,
+  imageUrl: PropTypes.string,
+  images: PropTypes.shape({
```

Comment:
What you think about making this prop accept a string or the object with this shape?
By doing that we have retro-compatibility. 
Another approach would be to use two props giving priority for the object one.

Another idea for the future is fallback the image from the object, otherwise, we will always have to pass all URLs

Reply:
Updated the PR to only show images if they’re available.